### DISCUSSION TITLE
image returned from the replicate api is a completely black image

### DISCUSSION THREAD

**Discussion by pijiuya** [ToTheBeginning/PuLID#31] [Q&A]
The image returned from the replicate api is a completely black image, but the background shows that the loading is successful.
![cb21b8941bf8be831a54f942ff3c347](https://github.com/ToTheBeginning/PuLID/assets/57893172/017d4814-7da0-41ef-b48e-decf47e668df)

**Comment by pijiuya** [comment]
![1715690527745](https://github.com/ToTheBeginning/PuLID/assets/57893172/699eda22-46ea-4916-bb2e-65ed712bf6fa)
They all seem to be successful, whether it is API or web, but without exception they are black

**Comment by zsxkib** [comment]
This was fixed btw
